### PR TITLE
added ns-password documentation

### PR DESCRIPTION
**Pull request by awebberCentrica**
Adding the ns-password documentation

### CODE REVIEW

File: docs/components/ns-password.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## Component relationship
+
+|  **Relationship**  |  |
+| :--- | :--- |
+| **Does it live in a panel?** | ❌ No |
+| **Does it live inside other components?** |  ✅ Yes -  `ns-form`|
+| **What layout classes can be used?**  | None |
+
+## Specification
+
```
Comment (dbenmore) [inline docs/components/ns-password.md:27]:
```suggestion
Also see ns-inputter [specification](components/ns-inputter.md#specification)

```

File: docs/components/ns-password.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+| Attribute | Type | Default | Options | Description |
+| :--- | :--- | :--- | :--- | :--- |
+| `value`   | `string` | `''` |  | The value of the input or group of inputs inside `ns-inputter`. This can be used to add an initial value |
+| `helper` | `string` |  |  | Adds a message between the label and the input. Used to convey a message to help the user to fill in the input |
```
Comment (dbenmore) [inline docs/components/ns-password.md:31]:
```suggestion
| `helper`       | `string`  |           |         | See [ns-inputter](components/ns-inputter.md#specification). |
```

File: docs/components/ns-password.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+| `name` | `string` |  |  | The name of this field as it will display in the form data. |
```
Comment (dbenmore) [inline docs/components/ns-password.md:32]:
```suggestion
| `name`         | `string`  | "password"    |         | See [ns-inputter](components/ns-inputter.md#specification). |
```

File: docs/components/ns-password.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+| Slots | Type |
+| :--- | :--- |
+| `tip-details` | `p tag` |
+
+## Specification notes
```
Comment (dbenmore) [inline docs/components/ns-password.md:38]: This section's not needed

**Review by dbenmore — APPROVED** [review]

**Review by andij — APPROVED** [review]

**Pull request merged**